source/documentation/deploying_services/postgres.md

```diff
@@ -170,19 +170,19 @@ Note that data restore will not be available in the event of an RDS outage affec
 
 For more details about how the RDS backup system works, see [Amazon's DB Instance Backups documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.BackingUpAndRestoringAmazonRDSInstances.html) [external page].
 
-### Restoring a PostgreSQL service snapshot
+### Restoring a PostgreSQL service snapshot (experimental)
 
 As a tenant, you can create a copy of any existing PostgreSQL service instance using the latest snapshot of the RDS instance.
 These snapshots are taken during [the nightly backups described above](#postgresql-service-backup).
 
 This can be useful if you want to clone a production database to be used for testing or batch processing.
 
-This feature currently has some limitations:
+This feature is experimental; we expect it to work and we'd like people who use it to tell us if it worked for them. It has the following limitations:
 
  * You can only restore the latest snapshot.
  * You cannot restore from a service instance that has been deleted.
  * You must use the same service plan for the copy as for the original service instance.
```

Comment:
Again, this wasn't part of this pull request, but it seems small, so I'll do it.